app/views/shared/_service_request_fields.html.slim

```diff
@@ -0,0 +1,68 @@
+.content-wrapper.category-wrapper.mt100
+  h1 What can we help you with?
+  h3 Select service to start.
+  = f.input :category_id, as: :radio_buttons, collection: categories[nil].map { |c| [ category_icons(c), c.id ]}, label: false 
+.content-wrapper.subcategories-wrapper.mt100.hidden
+  - @categories[nil].each do |category|
```

Comment:
I think it will be more correct to use local variable `categories` here instead of instance variable.